lambda/skill/i18n/language_model_it.py

```diff
@@ -0,0 +1,98 @@
+from datetime import tzinfo
+from skill.i18n.language_model_abc import LanguageModelABC
+
+
+class LanguageModelIT(LanguageModelABC):
+    def __init__(self, timezone: tzinfo):
+        self.timezone = timezone
+        self.SKILL_NAME = 'Telegram Connect'
+        self.SKILL_NAME_SPOKEN_IT = "<lang xml:lang='it-IT'>Telegram Connect</lang>".format(self.SKILL_NAME)
+
+        self.set_language_model()
+
+    def set_language_model(self):
+        ##############################
+        # GeneralStuff
+        ##############################
+        self.ACKS = ["Okay", "Va bene", "Okaaay"]
+        self.ACCEPTANCE_ACKS = ["Okay", "Sicuro", "Va bene", "Ricevuto", "You got it"]
+        self.DONE_ACKS = ["Okay", "Va bene", "Ricevuto", "Fatto", "You got it"]
+        self.THINKING = ["Umm", "Ahhm", "Hmmm"]
+        self.ANYTHING_ELSE = ["C'è altro?", "C'è altro che posso fare per te?",
+                              "Hai bisogno di altro?", "C'è altro di cui hai bisogno?"]
+        self.GOODBYES = ["Ciao", "Stammi bene", "Ciao ciao", "A dopo", "Ciaaao", "Passo e chiudo", "Abbi cura di te",
+                         "Arrivederci", "A più tardi", "A più tardi", "A presto", "Ci sentiamo dopo",
+                         "Alla prossima", "Divertiti", "Passa una buona giornata!"]
+        self.DONT_UNDERSTAND = ["Scusa", "Scusami", "Perdonami"]
+
+        self.GOOD_EVENING = "Buonasera!"
+        self.GOOD_MORNING = "Buongiorno!"
+        self.GOOD_AFTERNOON = "Buon pomeriggio!"
+        self.GOOD_BYE_EVENING = "Passa una buona serata!"
+        self.GOOD_BYE_MORNING = "Passa una buona giornata!"
+
+        self.FALLBACK = self.get_random_dont_understand() + ", cosa hai detto?"
+        self.EXCEPTION = "Si è verificato un errore inaspettato. Se hai un po di tempo, dai il tuo feedbeck su " \
+                         "GitHub. A presto."
+
+        ##############################
+        # LaunchRequestHandler
+        ##############################
+        self.WELCOME_BACK = 'Bentornato!'
+        self.NO_NEW_TELEGRAMS = "Non hai nuovi messaggi telegram"
+        self.NEW_TELEGRAMS = 'Hai nuovi messaggi telegram. Vuoi ascoltarli?'
+        self.NEW_SETUP = 'Benvenuto a {}. {} integra Alexa con Telegram ' \
+                         'Messenger. Sei pronto alla configurazione?'.format(self.SKILL_NAME_SPOKEN_IT,
```

Comment:
Ahh ok, I see that you changed it here as well. But as mentioned above: If you want that Alexa pronounces it with Italian accent you could just use `self.SKILL_NAME`.